### DISCUSSION TITLE
希望新增“麦克风模式”切换功能

### DISCUSSION THREAD

**Discussion by kinglost115** [seerge/g-helper#1332] [Ideas]
期待使用GHelper替换奥创中心，可是卸载奥创之后，想要切换麦克风模式就没法切换了。

“切换麦克风模式”指切换麦克风的收音方式

**Comment by seerge** [comment]
@kinglost115 hello, microphone modes can be controlled via Realtek panel, check this issue for more info https://github.com/seerge/g-helper/issues/18

**Comment by kinglost115** [comment]
I have checked the latest version of the realtek audio console software and there is indeed no option for microphone mode switching in the settings. Anyway, thank you for your response.
    
         
        
            
                        ***@***.***
            
        
     
        
  
    ---- 回复的原邮件 ----
  
  
    
      
        
         发件人 
        
        
            ***@***.***>
            
        
      
      
        
         发送日期 
        
        
        2023年9月20日 17:06
        
      
      
        
         收件人 
        
        
         
          
            ***@***.***>
            
          
        
      
      
        
         抄送人 
        
        
          
            ***@***.***>
            ,
          
          
            ***@***.***>
            
          
        
      
      
        
         主题 
        
        
              Re: [seerge/g-helper] 希望新增“麦克风模式”切换功能 (Discussion #1332)
        
      
    
  

@kinglost115 hello, microphone modes can be controlled via Realtek panel, check this issue for more info #18
